packages/sfc-playground/src/App.vue

```diff
@@ -54,7 +54,7 @@ const sfcOptions = computed(
     template: {
       isProd: productionMode.value,
       compilerOptions: {
-        isCustomElement: (tag: string) => tag === 'mjx-container',
+        isCustomElement: (tag: string) => tag === 'mjx-container' || tag.startsWith('my-'),
```

Comment:
This can be removed once reviews are complete, I just wanted to add a quick way to show the fix in the SFC playground for reviewers.